docs/creating-multinode-controlplane-clusters.md

```diff
@@ -0,0 +1,67 @@
+  - [Instances for the new machines not getting created](#instances-for-the-new-machines-not-getting-created)
+  - [Instances were created and initialized, but nodes don't show up on running 'kubectl get no'](#instances-were-created-and-initialized-but-nodes-dont-show-up-on-running-kubectl-get-no)
+
+## Prerequisites
+### Creating cluster with single instance controlplane.
+Following the [getting started guide](docs/getting-started.md), please set up a cluster with a single control plane instance.
+
+## Growing controlplane into multi-node controlplane.
+At this point, a cluster with a single instance control plane has been created and is responding to `kubectl`. Please refer to the 'Using the Cluster' section of the [getting started guide](docs/getting-started.md) for instructions. If you are unable to run `kubectl` against the cluster, please follow the 'Troubleshooting' section of the [getting started guide](docs/getting-started.md) to resolve the problem. 
+
+Now, that the cluster with the single instance control plane has been created, more controlplane instances can be added by creating a machine object for each desired controlplane machine, one machine at a time. 
+1. Copy the following yaml into a file, say `controlplane-machine.yaml`, and for each desired controlplane machine, update `controlplane-machine.yaml` with the machine and cluster name.
+   
+  ```bash
+  apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    name: <CONTROLPLANE_MACHINE_NAME>
+    labels:
+      cluster.k8s.io/cluster-name: <CLUSTER_NAME>
+      set: controlplane
+  spec:
+    versions:
+      kubelet: v1.13.2
+      controlPlane: v1.13.2
+    providerSpec:
+      value:
+        apiVersion: awsprovider/v1alpha1
+        kind: AWSMachineProviderSpec
+        instanceType: "t2.medium"
+        iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
+        keyName: "cluster-api-provider-aws.sigs.k8s.io"
+  ```
+  *Pro tip* 💡: You may refer to the machine's yaml in the release to create the above yaml for yourself.
+
+2. For each desired controlplane machine, create the machine object, alongside other machines of the cluster, by applying the `controlplane-machine.yaml`, from above.
+```bash
+kubectl apply -f controlplane-machines.yaml
+``` 
+Note: You may have to use the `-n <NAMESPACE>` option to kubectl incase your cluster and machines were created in the non-default namespace.
+
+3. This step is optional. Watch these newly created machine objects being reconciled by following the logs from the `aws-provider-controller-manager-0` pod using the below command
+```bash
+kubectl -n aws-provider-system logs -f aws-provider-controller-manager-0
+```
+Additionally, view the new instances initializing in the AWS console.
+
+4. Once the machine objects have been reconciled and the instances in AWS have been created and initialized, `kubectl get no` should show the two new machines with master roles.
```

Comment:
It would probably be best to avoid shortened nouns in the docs to avoid user confusion.
```suggestion
4. Once the machine objects have been reconciled and the instances in AWS have been created and initialized, `kubectl get nodes` should show the two new machines with master roles.
```